lib/server_communication/helper_methods.dart

```diff
@@ -11,14 +11,16 @@ import 'user_sync.dart';
 class HelperMethods {
   static const String scheme = "https";
   static const String host = "nssl.susch.eu";
+  static const int port = 80;
```

Comment:
bad port for productive usage, because the https port is 443
```suggestion
  static const int port = 443;
```